app/models/game.rb

```diff
@@ -1,5 +1,70 @@
 class Game < ApplicationRecord
-  belongs_to :white_player, :class_name => 'User'
-  belongs_to :black_player, :class_name => 'User'
+  belongs_to :white_player, :class_name => 'User', optional: true
+  belongs_to :black_player, :class_name => 'User', optional: true
   has_many :pieces
+  after_create :populate_game!
+
+  def populate_game!
+    [true, false].each do |is_white|
+      y = is_white ? 1 : 6
+      8.times do |x|
+        Pawn.create(
+          :game_id => self.id, 
+          :x_coord => x, 
+          :y_coord => y, 
+          :type => 'Pawn', 
```

Comment:
No need for type, since you are running the `Pawn.create(..`

Reply:
I'd suggest to bring the whole `create..` to one line - just for the sake of readability and to make the code more scannable...